src/index.ts

```diff
@@ -1,4 +1,17 @@
-export * from './sqlFormatter.js';
+export { supportedDialects, format, formatDialect } from './sqlFormatter.js';
+export { expandPhrases } from './expandPhrases.js';
+
+// Intentionally use "export *" syntax here to make sure when adding a new SQL dialect
+// we wouldn't forget to expose it in our public API.
+export * from './allDialects.js';
+
+// NB! To re-export types the "export type" syntax is required by webpack.
+// Otherwise webpack build will fail.
```

Comment:
there's an eslint-typescript rule here we can leverage, `use-consistent-type-exports`

Reply:
I can make a PR to update eslint rules soon

Reply:
Good find. I added the rule on my own. Seems to work just as needed.